src/main/java/org/data_bases_hwork/DataInsert.java

```diff
@@ -0,0 +1,134 @@
+package org.data_bases_hwork;
+
+
+import java.sql.*;
+import java.util.List;
+
+
+public class DataInsert {
+
+
+    static final String URL = "jdbc:postgresql://localhost:5432/databasebooking";
+    static final String USER = "postgres";
+    static final String PASS = "root";
+
+    private static final String INSERT_ACCOMMODATIONS_SQL = "insert into accommodation" +
+            "(id,type,bed_type,max_guests,description) values " + " (?,?,?,?,?);";
+
+    private static final String INSERT_ROOM_FAIR_SQL = "insert into room_fair" + "(id,value,season) VALUES " +
+            "(?,?,?);";
+
+    private static final String INSERT_ACCOMMODATION_ROOM_FAIR_RELATION = "insert into accommodation_room_fair_relation"
+            + "(id, accommodation_id, room_fair_id) VALUES " + "(?,?,?);";
+    final String format = "%20s%20s%12s\n";
+    Statement statement = null;
+    ResultSet resultSet = null;
+
+    public void insertAccommodation(List<Accommodation> accommodations) {
+        try (Connection connection = DriverManager.getConnection(URL, USER, PASS);
```

Comment:
you can add this to the class contructor since you will always need a connection

Reply:
also you can create a generic method to insert data and overload it for each type of data
insertData(`List <Acommodation>`)  and change the parameter type.